app/main.py

```diff
@@ -2,0 +1,20 @@
+from typing import Union
+from app.cafe import Cafe
+from app.errors import (
+    NotWearingMaskError,
+    VaccineError
+)
```

Comment:
Code Style: While importing multiple classes/modules, it's recommended to use parentheses `()` for better readability. Here is how you can do it:
```python
from app.errors import (
    NotWearingMaskError,
    VaccineError
)
```